custom_components/blitzortung/__init__.py

```diff
@@ -316,3 +211,111 @@ def is_connected(self):
     async def _tick(self, *args):
         for cb in self.on_tick_callbacks:
             cb()
+
+
+BlitzortungConfigEntry = ConfigEntry[BlitzortungCoordinator]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Initialize basic config of blitzortung component."""
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
It looks we don't need to store antything in `hass.data` now, right?

Reply:
We store yaml `config` in `hass.data` but this `config` has a defined type so we don't have to worry about creating a default dict.